Eevee.Sleep.Bot/Workers/Scrapers/InGameAnnouncement/OfficialSite/IndexScraper.cs

```diff
@@ -1,17 +1,17 @@
 using Eevee.Sleep.Bot.Enums;
 using Eevee.Sleep.Bot.Exceptions;
 using Eevee.Sleep.Bot.Extensions;
-using Eevee.Sleep.Bot.Models.InGameAnnouncement;
+using Eevee.Sleep.Bot.Models.InGameAnnouncement.Officialsite;
 using MongoDB.Driver.Linq;
 
-namespace Eevee.Sleep.Bot.Workers.Scrapers.InGameAnnouncement;
+namespace Eevee.Sleep.Bot.Workers.Scrapers.InGameAnnouncement.Officialsite;
```

Comment:
Possibly wrong folder structure, or maybe make it `Announcement.OfficialSite` / `Announcement.InGame` instead (same for the other implementations)